revisions-extended/src/plugins/revision-editor/revision-indicator/index.js

```diff
@@ -24,39 +24,42 @@ export const NOTICE_ID = 'revisions-extended-notice';
 
 const RevisionIndicator = () => {
 	const { savedPost } = usePost();
-	const { type: parentType, getLabel } = useParentPost();
+	const { type: parentType } = useParentPost();
+	const { loaded: loadedTypes, getTypeInfo } = useTypes();
 
 	const getRevisionType =
 		savedPost.status === POST_STATUS_SCHEDULED
 			? __( 'scheduled' )
 			: __( 'pending' );
 
-	const notes = [
-		sprintf(
-			// translators: %s: revision type.
-			__( 'You are currently editing a <b>%s update</b>.' ),
-			getRevisionType
-		),
-		sprintf(
-			// translators: %1$s: url %2$s: post type.
-			__( '[ <a href="%1$s">Edit %2$s</a>.' ),
-			getEditUrl( savedPost.parent ),
-			getLabel( 'singular_name' ).toLowerCase()
-		),
-		` | <a href="${ getCompareLink( savedPost.id ) }" />${ __(
-			'See changes'
-		) }</a> ]`,
-	];
-
 	useEffect( () => {
-		if ( ! parentType ) return;
+		if ( ! parentType || ! loadedTypes ) return;
+
+		const notes = [
+			sprintf(
+				// translators: %s: revision type.
+				__( 'You are currently editing a <b>%s update</b>.' ),
+				getRevisionType
+			),
+			sprintf(
+				// translators: %1$s: url %2$s: post type.
+				__( '[ <a href="%1$s">Edit %2$s</a>.' ),
+				getEditUrl( savedPost.parent ),
+				getTypeInfo(
+					`${ parentType }.labels.singular_name`
+				).toLowerCase()
+			),
+			` | <a href="/wp-admin/revision.php?revision=${
+				savedPost.id
+			}&gutenberg=true" />${ __( 'See changes' ) }</a> ]`,
```

Comment:
This isn't the right URL for diffing the changes of an update. [Should be](https://github.com/coreymckrill/revisions-extended/issues/5#issuecomment-777896613) something like `revision.php?page=compare-updates&revision_id={$revision_id}`. Maybe worth using `addQueryArg()` here?

Also, hardcoding the `/wp-admin/` part will break in some situations, especially multisite. [It looks like there's a function in Gutenberg to handle it.](https://github.com/WordPress/gutenberg/blob/004a32ec09d5b789be694d85725fe49cca385f08/packages/editor/src/components/post-last-revision/index.js#L19-L22)

Reply:
Yeah, that's fair. It looks like that's modified code but it's just been moved into the `useEffect`. 

That URL is already updated in trunk. 
https://github.com/coreymckrill/revisions-extended/blob/577cb1dcba2aaa892bddbc86c0c79b26bf23606f/revisions-extended/src/plugins/revision-editor/revision-indicator/index.js#L46

 However, it's not using the WP admin function. I'll look into the difference since `addQueryArg` seems to be working.